components/admin-panel/sections/ActivityLog/ActivityDescription.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { capitalize } from 'lodash';
+import { useIntl } from 'react-intl';
+
+import { ActivityDescriptionI18n } from '../../../../lib/i18n/activities';
+import formatMemberRole from '../../../../lib/i18n/member-role';
+import { getCollectivePageRoute } from '../../../../lib/url-helpers';
+
+import Link from '../../../Link';
+import LinkCollective from '../../../LinkCollective';
+import LinkExpense from '../../../LinkExpense';
+
+const ActivityDescription = ({ activity }) => {
+  const intl = useIntl();
+
+  if (!ActivityDescriptionI18n[activity.type]) {
+    return capitalize(activity.type.replace('_', ' '));
+  }
+
+  return intl.formatMessage(ActivityDescriptionI18n[activity.type], {
+    FromAccount: () => <LinkCollective collective={activity.fromAccount} openInNewTab />,
+    Account: () => <LinkCollective collective={activity.account} openInNewTab />,
```

Comment:
@Betree : Maybe add `title` for these as well, setting that to collective description if that exists (otherwise use `noTitle` prop). Otherwise this will default to the `collective.name` which is pretty useless I think given the name is already visible to the user and they don't expect to see the same name when hovering above it.